examples/data_collect/main.cpp

```diff
@@ -436,16 +430,16 @@ int main(int argc, char *argv[]) {
     // Wait until the warmup time is finished
     if (warmup_time > 0) {
         do {
-            frameType = "raw";
-            uint16_t *pRawFrame;
+            uint16_t *pFrame;
             status = camera->requestFrame(&frame);
             if (status != Status::OK) {
                 LOG(ERROR) << "Could not request frame!";
                 return 0;
             }
-            status = frame.getData(frameType, &pRawFrame);
+            status = frame.getData(frame_type, &pFrame);
```

Comment:
when frame_type is set to "full-frame", it will be passed to getData() which doesn't know about "full-frame". Instead, it knows about "frameData".

Reply:
Changed frame_type from full-frame to frameData in the begining. Now this won't be an issue.